web/index.html

```diff
@@ -117,18 +119,17 @@ <h2>Background</h2>
       Example of incorrect Python code output from DeepSeek Coder 6.7B (running on llama.cpp).
     </p>
     <p>
-      There are a number of ways we could approach solving this problem, but the simplest is to just
-      run the LLM multiple times for the same prompt and filter out any incorrect ones that fail
+      There are a number of ways we could mitigate the effect of hallucinations. 
+      We just naively temperature sample the LLM multiple times for the same prompt and filter out the incorrect ones that fail
       compilation or a test bench. This is known as
       <b>sampling + filtering</b> (S+F), and it provides a reasonable baseline against which to
       compare alternative solutions.
     </p>
     <p>
-      Another approach is, rather than relying on the LLM to directly generate code on its own, to
-      incorporate its predictions as part of a
+      Another approach is to search for the correct sequence of tokens using a
       <a href="https://en.wikipedia.org/wiki/Tree_(data_structure)" target="_blank">tree-based</a>
       search algorithm. Neither strategy is likely to prove definitive in solving the issue of LLM
-      accuracy, but we'll explore them in greater depth to learn exactly how and when they might be
+      accuracy, simply because the learned probabilities are miscalibrated. We'll explore them in greater depth to learn exactly how and when they might be
```

Comment:
What I meant to say was there is not much what we can do at inference time either with S+F or MCTS if the probabilites are bad. But maybe it is apparent. 

I can remove it but because there is a claim - "Neither strategy is likely to prove definitive" i wanted to add something.

Reply:
Just removing this.

Reply:
sure